src/Sulu/Bundle/PreviewBundle/Resources/config/services.xml

```diff
@@ -60,6 +60,10 @@
 
             <tag name="sulu.websocket.message.handler" dispatcher="admin" alias="sulu_preview.preview"/>
         </service>
+
+        <service id="sulu_preview.preview.voter" class="Sulu\Bundle\PreviewBundle\Security\PreviewVoter" public="false">
```

Comment:
this seems the wrong place :) you could simple register it in `preview_config`:

```
services:
    sulu_preview.preview.voter:
        class: Sulu\Bundle\PreviewBundle\Security\PreviewVoter
        public: false
        tag:
          - {name: "security.voter"}
```

then this service is only active in preview window